### PR TITLE
Show telemetry in report

### PR DESCRIPTION
**Pull request by ibalejandro**

### CODE REVIEW

File: src/services/TelemetryAPI.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            } catch (error) {
+                console.error(error);
+            }
+        }
+        return heartRateAndOxygenSaturation;
+    }
+
+    static async getBreathingFrequency(audioUrl, token) {
+        let breathingFrequency = {"breathingFrequency": "N/A"};
+        if (audioUrl !== '' && audioUrl !== undefined) {
```
Comment (jpvg10) [inline src/services/TelemetryAPI.js:34]:
```suggestion
        if (audioUrl) {
```

Comment (ibalejandro) [reply src/services/TelemetryAPI.js:34]: The thing is that the request should be triggered only if the audioUrl is different from ''. At the moment, the audioUrl is undefined.

Comment (jpvg10) [reply src/services/TelemetryAPI.js:34]: But as `''` is falsy, this will cover your case. Anyway, it's just a small detail.

Comment (ibalejandro) [reply src/services/TelemetryAPI.js:34]: Ok, I didn't know that. Let me do this change and thanks for the suggestion. It's cleaner.

Comment (jpvg10) [reply src/services/TelemetryAPI.js]: Great! 😄

**Pull request merged**